pkg/telemetry/prometheus/quality.go

```diff
@@ -0,0 +1,47 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/livekit/protocol/livekit"
+)
+
+var (
+	qualityRating prometheus.Histogram
+	qualityScore  prometheus.Histogram
+	qualityDrop   *prometheus.CounterVec
```

Comment:
This tracks the number of tracks that have a drop in quality in the last evaluation window. Note the above two are recorded at participant level. A participant quality is the weakest quality of all its tracks (published and subscribed). This counts number of tracks that has dropped in quality in the last evaluation window and is a vector to record upstream and downstream quality drops independently.